web/src/main/java/cz/muni/fi/pa165/security/SimpleGrantedAuthorityFactory.java

```diff
@@ -0,0 +1,17 @@
+package cz.muni.fi.pa165.security;
+
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import sun.java2d.pipe.SpanShapeRenderer;
+
+/**
+ * Created by martin on 11/12/16.
+ */
+public class SimpleGrantedAuthorityFactory {
+
+    private final static String ROLE_PREFIX = "ROLE_";
+
+    public static SimpleGrantedAuthority createAuthority(String role) {
```

Comment:
Hej hej to som mal na mysli aj ja, len som ti chcel povedat ze takto to nema zmysel ako to je zatial tu.